integrations/templates/metrics.md

```diff
@@ -0,0 +1,49 @@
+[% if entry.metrics.scopes %]
```

Comment:
We need [metrics. description](https://github.com/netdata/go.d.plugin/blob/42a9c4dae22ada85074a2ddb5ab05867b039f996/modules/prometheus/metadata.yaml#L228-L247) even when there are no scoped.

Reply:
Ok, it didn't work on Fri (prem had empty "Metrics"). It works now, I see.